app/web_framework/cluster_base_handler.py

```diff
@@ -62,6 +64,31 @@ def _handle_request_exception(self, ex):
         self.set_status(status_code)
         self.finish()
 
+    # For methods that annotate with `Tuple`, use `disable=invalid-sequence-index`
+    # This bug is fixed in https://github.com/PyCQA/astroid/commit/563031aaf13a44adc8db4f8d0ab8020d550aae00
+    # More information on the issue in https://github.com/PyCQA/pylint/issues/1212
+
+    def get_pagination_params(self) -> Tuple[int, int]:  # pylint: disable=invalid-sequence-index
+        offset = self.get_query_argument('offset', Configuration['pagination_offset'], True)
+        limit = self.get_query_argument('limit', Configuration['pagination_limit'], True)
+        try:
+            offset = int(offset)
+            limit = int(limit)
+        except ValueError as ex:
+            raise BadRequestError from ex
+        return self._validate_pagination_params(offset, limit)
+
+    @staticmethod
+    def _validate_pagination_params(offset: int, limit: int) -> Tuple[int, int]:  # pylint: disable=invalid-sequence-index
```

Comment:
This doesn't look like it validates anything. In fact, it looks like it does exactly the opposite of validation because it silently fails over to default values. I suggest renaming it to something like _get_pagination_params_or_safe_defaults. Something to indicate that it won't raise an error and that it will return sensible defaults. 

Honestly, if you just combined the two methods, that would would work too.